include/openflow/orange-ext.h

```diff
@@ -40,4 +40,14 @@ struct ol_load_filter_prog {
 };
 OFP_ASSERT(sizeof(struct ol_load_filter_prog) == 8);
 
+struct ol_update_map {
+    ovs_be16 filter_prog;  /* Filter program ID. */
+    ovs_be16 map_id; /* Map ID. */
+    ovs_be32 key_size;
+    ovs_be32 value_size;
+    /* Followed by:
+     *   - Exactly (key_size + value_size) bytes. */
+    /* uint8_t entries[...]; */ /* Data containing the map entries (key + value). */
```

Comment:
Could we make this into a batch operation? I.e., allowing users to update several entries at the same time. We'll need an additional `nb_elems` field for that.

Reply:
Honestly, I'd prefer to keep such a design. This design follows the same approach as in the bpftool, where the batch operation is not allowed. What do you think?

Reply:
It makes sense for a command-line tool because the syntax would quickly become cumbersome, but here we primarily are designing a network message. Are there no situations were you might need to update several key-value couples? I'd rather avoid having one OpenFlow message per key-value couple. We can prohibit multiple couples for `ovs-ofctl` though.

If you don't see a need for this, we can always do it later. It's a research prototype after all, so we don't need to have a fixed API.

Reply:
Actually, if you don't see a need for this, could you still add a `nb_elems` field to the structure (its value doesn't really matter). That could save us some time later if we want to add support as it'll be backward-compatible.

Reply:
> It makes sense for a command-line tool because the syntax would quickly become cumbersome, but here we primarily are designing a network message. Are there no situations were you might need to update several key-value couples? 

Ok, this argument convienced me :) I didn't consider it from the OpenFlow protocol perspective.  I will add nb_elems field to the structure and try to find time to implement the handling code.